transbank-sdk-dotnet-rest-example/Controllers/TransaccionCompletaController.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Mvc;
+using Transbank.Common;
+using Transbank.Webpay.TransaccionCompleta;
+
+namespace transbanksdkdotnetrestexample.Controllers
+{
+    public class TransaccionCompletaController : Controller
+    {
+
+        public ActionResult Create()
+        {
+            var random = new Random();
+
+            var buy_order = random.Next(999999999).ToString();
```

Comment:
Is `snake_casing` really used on C#? 

I understand that `camelCase` is the usual convention and this code looks _very_ weird when mixing casing below.